README.md

```diff
@@ -33,6 +33,8 @@ make[1]: Leaving directory '.../linux-with-rust-support'
 
 For details about the Rust support, see https://rust-for-linux.com.
 
+For details about getting started with kernel development in rust, see https://docs.kernel.org/rust/quick-start.html
```

Comment:
I think it may be best to link the higher level page, which already links the Quick Start guide, e.g. what about:

```suggestion
For details about getting started with kernel development in Rust, see https://docs.kernel.org/rust/.
```